.appveyor/install.cmd

```diff
@@ -0,0 +1,64 @@
+@echo off
+setlocal enableextensions enabledelayedexpansion
+	cinst wget
+
+	if not exist "%PHP_BUILD_CACHE_BASE_DIR%" (
+		echo Creating %PHP_BUILD_CACHE_BASE_DIR%
+		mkdir "%PHP_BUILD_CACHE_BASE_DIR%"
+	)
+
+	if not exist "%PHP_BUILD_OBJ_DIR%" (
+		echo Creating %PHP_BUILD_OBJ_DIR%
+		mkdir "%PHP_BUILD_OBJ_DIR%"
+	)
+
+	if not exist "%PHP_BUILD_CACHE_SDK_DIR%" (
+		echo Cloning remote SDK repository
+		rem git clone -q --depth=1 --branch %SDK_BRANCH% %SDK_REMOTE% "%PHP_BUILD_CACHE_SDK_DIR%" 2>&1
+		git clone --branch %SDK_BRANCH% %SDK_REMOTE% "%PHP_BUILD_CACHE_SDK_DIR%" 2>&1
```

Comment:
Why does the commented command above have `-q --depth=1`. You use those arguments for cloning PHP below, so why not for the SDK as well?

Reply:
Snippet I got from Anatol. I guess the ``rem`` line can go.

Reply:
I'd suggest adding `-q --depth=1` to this `git clone`, though. I don't see any reason you need the full commit history, and `-q` will make the output a bit quieter. Since progress won't be reported to stderr, we can probably remove the redirection with `2>&1` as well.